src/Makefile

```diff
@@ -3126,6 +3126,61 @@ mz2500_tiny_buffered:
 	mv a.2D $(BUILD_PATH)/TINY_mz2500_buffered.2D
 	rm a.bin
 	
+test:
+	$(Z88DK_PATH)$(MYZ88DK) +cpm \
+	-SO3 \
+	-compiler=sdcc \
+	-DFORCE_XSIZE=16 \
+	-DFORXE_YSIZE=16 \
+	-DFORCE_GHOSTS_NUMBER=6 \
+	-DFORCE_BOMBS_NUMBER=2 \
+	-vn \
+	-DNO_COLOR \
+	-DTINY_GAME \
+	-DNO_BLINKING \
+	-DNO_CONTROL_INSTRUCTIONS \
+	-DNO_DEAD_GHOSTS \
+	-DNO_SLEEP \
+	-DNO_INITIAL_SCREEN \
+	-DNO_SET_SCREEN_COLORS \
+	-lndos \
+	-DBUFFERED -DNO_INIT_GRAPHICS \
+	-DFORCE_NARROW \
+	-DNO_STATS \
+	-DALT_PRINT \
+	-DALT_CLEAR_SCREEN \
+	-DSTDLIB \
+	-DNO_WAIT \
+	-DBUFFRED \

```

Comment:
Thanks! Lots of typos. I have fixed most of them and I am going to fixed more. 
I am using lots of options with the MZ2500 because Z88DK barely supports it with several constraints on size of ram and of the disk image and total lack of graphics libraries.